src/components/Dashboard.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div>
+  <div>
+  <MySidebar></MySidebar>
+</div>
+<div class="main">
+       <div class="card-body text-center">
+    <h4 class="display-4">Configurations Dashboard</h4>
+  </div>
+
+       <table class="table table-striped table-bordered ">
+  <thead class="thead-dark">
+    <tr>
+      <th scope="col"><p align="center">ID</p></th>
```

Comment:
I generally like idea to have Primary Key.
But you have to move your perspective now as a "dummy" user of such system. What this info, this unique primary key from our system, brings to "end-user"? Does it somehow helps him? I don't think so.


Anyway, as I am not strict to this one, this is just learning stuff. So maybe for the first version you might leave it.